### PR TITLE
Add script that will clean up extraneous posts we received

### PR DESCRIPTION
**Pull request by sbsmith86**
#### What's this PR do?

This PR adds a command `php artisan rogue:postcleanup {date}` that will go through all the posts we received on the given date, find the duplicate submissions (by checking for duplicate `captions`) and then delete all but the most recent submission from the DB. 

#### How should this be reviewed?

👀 

Definitely double check the deletion logic: 

I used `$post->forceDelete()` to truly delete the record from the DB because we don't want a record of these bad posts. 

I used `Storage::delete({path_to_file})` to delete the image from storage. 

#### Any background context you want to provide?

This script is meant to clean up the duplicate posts we receive because of this bug: 
https://github.com/orgs/DoSomething/teams/team-bleed/discussions/5

#### Relevant tickets
Fixes 🐛 

#### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Tested on staging.
- [ ] Pinged a PM if this is a larger PR that would benefit from some additional testing love.

### CODE REVIEW

File: app/Console/Commands/PostCleanup.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            ->having('caption_count', '>', 1)
+            ->get();
+
+        $this->info('There are ' . $postsByCaption->count() . ' posts with more than 1 of the same caption');
+
+        // Loop through all the posts that have the
+        // same caption per signup and delete all but the most recent one.
+        foreach ($postsByCaption as $postCaption) {
+            $this->info('Getting posts under signup ' . $postCaption->signup_id);
+            $userPosts = Post::where('signup_id', $postCaption->signup_id)
```
Comment (katiecrane) [inline app/Console/Commands/PostCleanup.php]: I'm trying to work through this and this part seems to me like it is deleting all posts under that signup on that day except for the most recent one. What if a user had submitted multiple posts with different captions?

Comment (sbsmith86) [reply app/Console/Commands/PostCleanup.php]: And then I go through each of those `signup_id` and get all those posts and delete all but the most recent one.

Comment (sbsmith86) [reply app/Console/Commands/PostCleanup.php]: Yea, I guess that is a possibility but I wasn't really seeing that. I can add that extra filter!

Comment (katiecrane) [reply app/Console/Commands/PostCleanup.php]: Woo! Then we can 🔥 🔥 🔥 !!

**Comment by sbsmith86** [comment]
@katiecrane yea i figure dump some fake posts in there and then run this to make sure they get cleaned up.

**Pull request merged**